Enable to alter settings in subscribed tables (#12523)
crate/crate: docs/appendices/release-notes/unreleased.rst

Changes
=======

> - Enabled to allow users to alter the settings ``number_of_replicas`` and
>   ``refresh_interval`` for tables subscribed in a logical replication.

Reviewer: ```suggestion
- Enabled users to alter the settings ``number_of_replicas`` and
  ``refresh_interval`` on tables subscribed in a logical replication.
```
